2023-07-20 15:48:55,744 INFO    MainThread:13169 [internal.py:wandb_internal():90] W&B internal server running at pid: 13169, started at: 2023-07-20 15:48:55.743777
2023-07-20 15:48:55,745 DEBUG   HandlerThread:13169 [handler.py:handle_request():141] handle_request: run_start
2023-07-20 15:48:55,746 INFO    WriterThread:13169 [datastore.py:open_for_write():75] open: /home/nicole/marl-emecom/src/experiments_pgg_v0/anastassacos/wandb/offline-run-20230720_154855-2t62fnkp/run-2t62fnkp.wandb
2023-07-20 15:48:55,746 DEBUG   HandlerThread:13169 [meta.py:__init__():35] meta init
2023-07-20 15:48:55,746 DEBUG   HandlerThread:13169 [meta.py:__init__():49] meta init done
2023-07-20 15:48:55,746 DEBUG   HandlerThread:13169 [meta.py:probe():209] probe
2023-07-20 15:48:55,747 DEBUG   HandlerThread:13169 [meta.py:_setup_git():199] setup git
2023-07-20 15:48:55,755 DEBUG   HandlerThread:13169 [meta.py:_setup_git():206] setup git done
2023-07-20 15:48:55,755 DEBUG   HandlerThread:13169 [meta.py:_save_pip():53] save pip
2023-07-20 15:48:55,756 DEBUG   HandlerThread:13169 [meta.py:_save_pip():67] save pip done
2023-07-20 15:48:55,756 DEBUG   HandlerThread:13169 [meta.py:probe():247] probe done
2023-07-20 15:49:00,894 WARNING MainThread:13169 [internal.py:wandb_internal():153] Internal process interrupt: 1
2023-07-20 15:49:01,052 WARNING MainThread:13169 [internal.py:wandb_internal():153] Internal process interrupt: 2
2023-07-20 15:49:01,052 ERROR   MainThread:13169 [internal.py:wandb_internal():156] Internal process interrupted.
2023-07-20 15:49:01,749 INFO    SenderThread:13169 [sender.py:finish():1111] shutting down sender
2023-07-20 15:49:01,895 INFO    WriterThread:13169 [datastore.py:close():279] close: /home/nicole/marl-emecom/src/experiments_pgg_v0/anastassacos/wandb/offline-run-20230720_154855-2t62fnkp/run-2t62fnkp.wandb
2023-07-20 15:49:01,895 INFO    HandlerThread:13169 [handler.py:finish():806] shutting down handler
2023-07-20 15:49:01,896 INFO    MainThread:13169 [internal.py:handle_exit():80] Internal process exited
